eduiddashboard/views/__init__.py

```diff
@@ -57,7 +57,8 @@ class BaseFormView(FormView):
 
     def __init__(self, context, request):
         super(BaseFormView, self).__init__(request)
-        self.user = context.user
+        self.user = get_session_user(self.request)
+        retrieve_modified_ts(self.user, request.dashboard_userdb)
```

Comment:
You are right. I thought it was a posterior PR, I got a bit lost moving forwards and backwards in the history of the various branches.